doc/list.rst

```diff
@@ -2055,6 +2055,23 @@ List of Available Rules
    Part of rule sets `@PhpCsFixer <./ruleSets/PhpCsFixer.rst>`_ `@Symfony <./ruleSets/Symfony.rst>`_
 
    `Source PhpCsFixer\\Fixer\\Operator\\OperatorLinebreakFixer <./../src/Fixer/Operator/OperatorLinebreakFixer.php>`_
+-  `ordered_attributes <./rules/attribute_notation/ordered_attributes.rst>`_
+
+   Sorts attributes using the configured sort algorithm.
+
+   Configuration options:
+
+   - | ``attributes_order``
+     | A list of FQCNs of attributes defining the desired order used by the custom sort algorithm.
+     | Allowed types: ``string[]``
+     | Default value: ``[]``
+   - | ``sort_algorithm``
```

Comment:
to be consistent with https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/phpdoc/phpdoc_order.rst and https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/class_notation/ordered_types.rst params naming the names should be:
- sort_algorithm: 'alpha'/'none'
- order: string[]

Reply:
That doesn't make sense since `none` is not an option in this fixer.

Reply:
so what is a `custom`?

Reply:
It allows you to provide a list of FQCNs of attributes defining the desired order.

Reply:
And when I want alpha order + some attributes with custom order/on top?

Reply:
Well, with the current implementation, it's either one or the other :smile: . If ever needed, the current functionality can be easily be extended.

Reply:
I wanted to point out 'alpha'/'none' sort might be better with the explicit order always put on top.

Reply:
Oh, ok, I get what you're saying now, yes that actually does make more sense when I think about it, either a sort algorithm or an explicit order. Tnx, I'll update the PR.